src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/CatchParameterNameCheck.java

```diff
@@ -56,12 +56,31 @@
  * An example of how to configure the check for names that begin with a lower case letter,
  * followed by any letters or digits is:
  * </p>
+ * <p>Configuration:</p>
  * <pre>
  * &lt;module name="CatchParameterName"&gt;
  *   &lt;property name="format" value="^[a-z][a-zA-Z0-9]+$"/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>Code example:</p>
+ * <pre>
+ * public class MyTest {
+ *   public void myTest() {
+ *     try {
+ *       // ...
+ *     } catch (ArithmeticException ex) { //OK
+ *       // ...
+ *     } catch (ArrayIndexOutOfBoundsException ex2) { //OK
+ *       // ...
+ *     } catch (IOException thirdException) { //OK
+ *       // ...
+ *     } catch (Exception FourthException) { // violation, the initial letter
+ *                                           // should match the pattern [a-z]
```

Comment:
please restore comment on how it was - "... lowercase".
"pattern" usage is confusing as it might be referencing of Check property.